sqlDocker/Dockerfile

```diff
@@ -1,17 +1,25 @@
 FROM mcr.microsoft.com/mssql/server:2017-latest
+
 # Install essentials
 RUN apt update
-RUN apt install -y \
-    build-essential 
+RUN apt install -y build-essential python3 python3-pip
+
 # Create a config directory
 RUN mkdir -p /usr/config
 WORKDIR /usr/config
 
 # Bundle config source
 COPY . /usr/config
 
-# Grant permissions for to our scripts to be executable
+# Install Python dependencies
+RUN pip3 install --no-cache-dir pandas
+
+# Copy the Python script inside Docker
+COPY finwire_parsing.py /usr/config/data/gendata/
```

Comment:
we already do `COPY . /usr/config` so this one is redundant

Reply:
Here you are refering to only the copy of the file right? Because I still need to do pip install for pandas (I've checked and when I try to call the script without installing the library, it returns errors).

On the other hand, I think I understand the idea behind the copy, so I'll just make sure that with the new folder and files distribution the parsing and load can be generated :)